fiftyone/server/metadata.py

```
@@ -77,6 +77,14 @@ async def get_metadata(
     Returns:
         metadata dict
     """
+    media_reference = sample.get("_media_reference")
+    if media_reference is not None:
+        cache_key = media_reference["key"]
+        if cache_key not in metadata_cache:
+            metadata_cache[cache_key] = dict(aspect_ratio=1)
```

Ruff (0.16.2)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)